packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -17,6 +17,16 @@ import 'typography.dart';
 
```

Comment:
looks like DateTime is always either in UTC or in "the local time zone". We use the date of the value in whatever time zone that DateTime is in. (e.g. `DateTime.parse('1970-01-01T00:00:00+0500')` generates a UTC date, so the year would be 1969.)